kai/rpc_server/server.py

```diff
@@ -213,6 +226,33 @@ def set_config(
         )
 
 
+@app.add_request(method="analysis_engine.Analyze")
+def doAnalyze(
+    app: KaiRpcApplication, server: JsonRpcServer, id: JsonRpcId, params: dict[str, Any]
+) -> None:
+    if not app.initialized:
+        server.send_response(id=id, error=ERROR_NOT_INITIALIZED)
+        return
+
+    try:
+        server.send_response(
+            id=id,
+            result=app.analyzer.run_analyzer_lsp(
```

Comment:
data class this result?

Reply:
I do not know that I fully understand what you are asking me to do. Should I be mirroring the `[]konveyor.Ruleset` types in kai and type checking the response I get from the analyzer?

Reply:
Ok ignore my original thought about data class because I forgot this is returning the ruleset types. However, the run_analyzer_lsp function can return either a response or an error. Setting the result field will always make it a valid response even if the actual thing we receive from analyzer is an error.

Reply:
Done.